resteasy-client/src/main/java/org/jboss/resteasy/client/jaxrs/internal/proxy/ClientInvoker.java

```diff
@@ -157,13 +160,31 @@ protected Object invokeSync(Object[] args)
    protected ClientInvocation createRequest(Object[] args)
    {
       WebTarget target = this.webTarget;
-      for (int i = 0; i < processors.length; i++)
+
+      Annotation[] pathParams = new Annotation[method.getParameterTypes().length];
+      HashMap<String, Object> pathParamsMap = new HashMap<>();
+
+      for (int i = 0; i < pathParams.length; i++)

```

Comment:
This could just be `method.getParameterTypes().length` and the `pathParams` could be deleted as it's only used here.

Reply:
Thank you for your advice, you are correct. I changed the code, tests run just fine and will push the changes once we resolve the second problem

Reply:
Fixed in new commit